package.json

```diff
@@ -72,6 +72,7 @@
   "homepage": "https://github.com/posva/vuepress-plugin-example-preview#readme",
   "dependencies": {
     "focus-visible": "^4.1.5",
-    "raw-loader": "^0.5.1"
+    "raw-loader": "^0.5.1",
+    "webpack": "^4.22.0"
```

Comment:
is this intentional? 🤔

Reply:
Forgot to remove it, it's just a test code and check if the latest webpack has fixed it.